packages/edit-site/src/components/site-hub/style.scss

```diff
@@ -42,22 +17,22 @@
 	}
 }
 
-.edit-site-site-hub__text-content {
-	// Necessary for the ellipsis to work.
-	overflow: hidden;
-}
-
 .edit-site-site-hub__title {
 	text-overflow: ellipsis;
 	white-space: nowrap;
 	overflow: hidden;
-}
-
-.edit-site-site-hub__site-title {
 	flex-grow: 1;
 	color: $gray-200;
 }
 
+.edit-site-site-hub__site-view-link {
+	flex-grow: 0;
+	margin-right: var(--wp-admin-border-width-focus);
```

Comment:
Not quite sure what this does, overall the search and view site links seem differently positioned here than in trunk.

But since they're wrong both here and in trunk (search icon should vertically align with chevrons below) then it shouldn't block this PR.

Reply:
I don't know, this style was already here, I just removed a wrapper (specificity) which seemed useless.

Reply:
Icon alignment is generally a mess in the sidebar. It would be good to tidy that up.